src/app/public/test-utility/page-object.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  Type
+} from '@angular/core';
+
+import {
+  ComponentFixture
+} from '@angular/core/testing';
+
+import {
+  By
+} from '@angular/platform-browser';
+
+export class PageObject<C> {
+  public constructor(private fixture: ComponentFixture<C>) {
```

Comment:
actually, this should be `public` or `protected` that way those extending this class can have access to it for whatever they might need.